src/pages/LoginProcess/AddProfilePicture/AddProfilePicture.jsx

```diff
@@ -2,41 +2,61 @@ import "./AddProfilePicture.css";
 import { useNavigate, useParams } from "react-router-dom";
 import Button from "../../../components/shared/Button/Button";
 import TopbarSignup from "../../../components/TopbarSignup/TopbarSignup";
-const AddProfilePicture = () => {
-	const navigate = useNavigate();
-	const { username } = useParams();
+import Parse from "parse/dist/parse.min";
+import React, { useEffect, useRef, useState } from "react";
+function AddProfilePicture() {
+  const navigate = useNavigate();
+  const { username } = useParams();
+  const [image, setImage] = useState("/icons/addProfile.svg");
+  const fileInputRef = useRef(null);
 
-	const handleSubmit = (e) => {
-		e.preventDefault();
-		// Add logic to handle account creation
-		navigate(`/added-picture/${username}`);
-	};
+  const handleImageChange = async (event) => {
+    if (event.target.files && event.target.files[0]) {
+      const file = event.target.files[0];
+      setImage(URL.createObjectURL(file));
 
-	return (
-		<div className="verification-page">
-			<div>
-				<TopbarSignup
-					prevProgress={0.6}
-					progressMax={0.8}
-				/>
+      try {
+        const parseFile = new Parse.File(file.name, file);
+        await parseFile.save();
```

Comment:
Same comment as before

Reply:
And maybe if it is possible could we also reuse the ProfilePicture2 component here?

Reply:
Can you specify which comment from before you mean? 

Sure, I can reuse ProfilePicture2 here but then it won't follow the Figma design.